### PR TITLE
jt fixes

### PR DESCRIPTION
**Pull request by eregon**

### CODE REVIEW

File: tool/jt.rb [hunk 1]
```diff
@@ -340,11 +333,22 @@ def replace_env_vars(string, env = ENV)
...
 
   def sh(*args)
-    Dir.chdir(TRUFFLERUBY_DIR) do
+    chdir(TRUFFLERUBY_DIR) do
       raw_sh(*args)
     end
   end
 
+  def chdir(dir, &block)
+    raise LocalJumpError unless block_given?
```
Comment (pitr-ch) [inline tool/jt.rb]: this might be confusing without any message. Argument error might be better fit.

Comment (eregon) [reply tool/jt.rb]: It's the typical error on missing block, like `yield` generates. I'll add a message.

Comment (eregon) [reply tool/jt.rb]: Fixed.

Comment (pitr-ch) [reply tool/jt.rb]: yeah but methods customary check explicitly for the block and raise something more friendly

**Pull request merged**